java/com/facebook/crypto/streams/NativeGCMCipherInputStream.java

```diff
@@ -136,6 +136,6 @@ public synchronized void reset() throws IOException {
 
   @Override
   public long skip(long byteCount) throws IOException {
-    throw new UnsupportedOperationException();
+    return super.skip(byteCount);
```

Comment:
So you'd go for the custom implementation of skip that does strictly what's needed? Sounds good.